content/changelog/index.um

```diff
@@ -8,21 +8,21 @@
     @groupByApi true
     @reverseVisibleList true
     @process
-      @inline ../docs/**/index.um
+      @inline ../docs/**/api.um
```

Comment:
Makes sense to only include the `api` sections here (reduces the amount of stuff we have to process)